specs/phase1/beacon-chain.md

```diff
@@ -943,7 +970,7 @@ def apply_shard_transition_updates(state: BeaconState,
         proposer_index = get_shard_proposer_index(state, slot, shard)
         proposer_reward = (
             get_base_reward(state, proposer_index)
-            * get_online_validator_indices(state)
+            * len(get_online_validator_indices(state))
             // get_active_shard_count(state)
         )
```

Comment:
Note: this code is creating a new class of reward (shard proposer reward) that's as large as a base reward in total size. Should consider (i) decreasing, and (ii) keeping it in mind when computing the base reward denominator.